platform/ios/src/UIViewController+MGLAdditions.h

```diff
@@ -0,0 +1,11 @@
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface UIViewController (MGLAdditions)
+
++ (UIViewController *)getTopViewControllerFrom:(UIViewController *)viewController;
```

Comment:
Nit: Prefixed method names on framework classes.
Btw, do we need the viewController arg when we have access to self?
`+[UIViewController(MGLAdditions) topViewController]` conflicts with `-[UINavigationViewController topViewController]` but what about `+[UIViewController(MGLAdditions) mgl_topMostViewController]`?